js/src/data/test/reducer.test.js

```diff
@@ -780,18 +795,51 @@ describe( 'reducer', () => {
 		// prettier-ignore
 		const argumentsTuples = [
 			[ TYPES.RECEIVE_SETTINGS, 'settings', 'mc.settings' ],
-			[ TYPES.RECEIVE_ACCOUNTS_JETPACK, 'account', 'mc.accounts.jetpack' ],
+			[
+				TYPES.RECEIVE_ACCOUNTS_JETPACK,
+				'account',
+				'mc.accounts.jetpack',
+			],
 			[ TYPES.RECEIVE_ACCOUNTS_GOOGLE, 'account', 'mc.accounts.google' ],
-			[ TYPES.RECEIVE_ACCOUNTS_GOOGLE_ACCESS, 'data', 'mc.accounts.google_access' ],
+			[
+				TYPES.RECEIVE_ACCOUNTS_GOOGLE_ACCESS,
+				'data',
+				'mc.accounts.google_access',
+			],
 			[ TYPES.RECEIVE_ACCOUNTS_GOOGLE_MC, 'account', 'mc.accounts.mc' ],
-			[ TYPES.RECEIVE_ACCOUNTS_GOOGLE_MC_EXISTING, 'accounts', 'mc.accounts.existing_mc' ],
-			[ TYPES.RECEIVE_ACCOUNTS_GOOGLE_ADS_BILLING_STATUS, 'billingStatus', 'mc.accounts.ads_billing_status' ],
-			[ TYPES.RECEIVE_ACCOUNTS_GOOGLE_ADS_EXISTING, 'accounts', 'mc.accounts.existing_ads' ],
+			[
+				TYPES.RECEIVE_ACCOUNTS_GOOGLE_MC_EXISTING,
+				'accounts',
+				'mc.accounts.existing_mc',
+			],
+			[
+				TYPES.RECEIVE_ACCOUNTS_GOOGLE_ADS_BILLING_STATUS,
+				'billingStatus',
+				'mc.accounts.ads_billing_status',
+			],
+			[
+				TYPES.RECEIVE_ACCOUNTS_GOOGLE_ADS_EXISTING,
+				'accounts',
+				'mc.accounts.existing_ads',
+			],
 			[ TYPES.RECEIVE_MC_CONTACT_INFORMATION, 'data', 'mc.contact' ],
-			[ TYPES.RECEIVE_TARGET_AUDIENCE, 'target_audience', 'mc.target_audience' ],
-			[ TYPES.SAVE_TARGET_AUDIENCE, 'target_audience', 'mc.target_audience' ],
+			[
+				TYPES.RECEIVE_TARGET_AUDIENCE,
+				'target_audience',
+				'mc.target_audience',
+			],
+			[
+				TYPES.SAVE_TARGET_AUDIENCE,
+				'target_audience',
+				'mc.target_audience',
+			],
 			[ TYPES.RECEIVE_MC_SETUP, 'mcSetup', 'mc_setup' ],
-			[ TYPES.RECEIVE_MC_PRODUCT_STATISTICS, 'mcProductStatistics', 'mc_product_statistics' ],
+			[
+				TYPES.RECEIVE_MC_PRODUCT_STATISTICS,
+				'mcProductStatistics',
+				'mc_product_statistics',
+			],
```

Comment:
It'd be great if we could revert these changes to the previous format for a bit better readability and also to conform to the [code comment above](https://github.com/songchen2022/google-listings-and-ads/blob/e9e5ec823168b40908e61cea8fb2ef2e80ddba31/js/src/data/test/reducer.test.js#L793-L795).